### PR TITLE
Rent return book

### PR DESCRIPTION
**Pull request by DragisaDraskovic**
Rent and return book task

### CODE REVIEW

File: src/components/bookDetails/BookDetails.css [hunk 1]
```diff
@@ -1,27 +1,191 @@
...
+    border-right: 1px solid #f8f8f8;
+    font-size: 12px;
+}
+
+.table_container thead th {
+    color: #ffffff;
+    background-color: #4FC3A1;
+}
+
+.table_container thead th:nth-child(odd) {
```
Comment (SaleBojke) [inline src/components/bookDetails/BookDetails.css:128]: 👍

File: src/components/bookDetails/BookDetails.tsx [hunk 2]
```diff
@@ -30,10 +39,10 @@ const BookDetails = () => {
       Quantity: 0
     }
   )
-
   useEffect(() => {
     getBook()
-  }, [])
+    getHistoryBooks()
```
Comment (SaleBojke) [inline src/components/bookDetails/BookDetails.tsx:44]: Do not fetch books history unless role is admin or librarian.

File: src/components/bookDetails/BookDetails.tsx [hunk 3]
```diff
@@ -66,35 +91,103 @@ const BookDetails = () => {
     setIsOpenEditModal(false)
   }
 
+  const handleRentBook = () => {
+    RentalServices.rentBook(bookDetails.Id)
+      .then(() => {
+        toast.success('Success rent book', {
+          position: toast.POSITION.TOP_RIGHT
```
Comment (SaleBojke) [inline src/components/bookDetails/BookDetails.tsx:98]: default position for message is top right, remove this property

File: src/components/bookDetails/BookDetails.tsx [hunk 4]
```diff
@@ -66,35 +91,103 @@ const BookDetails = () => {
...
+            <p>{bookDetails.Description}</p>
+          </div>
+          <h4>Authors:</h4>
+          <div className='container_author'>
+            {bookDetails.Authors &&
+            bookDetails.Authors.map((author) => ( <p key={author.Id} className='p_tag_for_author'>{author.Firstname} {author.Lastname} </p>))
+            }
+          </div>
+        </div>
+        { !isUser
```
Comment (SaleBojke) [inline src/components/bookDetails/BookDetails.tsx:148]: i would prefer is admin or librarian over not user

File: src/components/bookDetails/BookDetails.tsx [hunk 5]
```diff
@@ -66,35 +91,103 @@ const BookDetails = () => {
...
+        { isOpenEditModal && <ModalForEdit onClose={handleCloseModal} bookId={Number(id)} />}
+        <ToastContainer />
       </div>
-      <div className='container_for_button'>
-      <button className='button_edit' onClick={handleEditBook}>Edit</button>
-        <button className='button_delete' onClick={handleDeleteBook}>Delete</button>
-        <button className='button_rent'>Rent</button>
-        <button className='button_return'>Return</button>
-        <button className='button_back'>Back</button>
+      { !isUser
```
Comment (SaleBojke) [inline src/components/bookDetails/BookDetails.tsx:165]: same

File: src/components/modal/ModalCreateBooks.tsx [hunk 6]
```diff
@@ -148,6 +152,7 @@ const ModalCreateBooks = ({ onClose, bookId } : Modal) => {
               placeholder='ISBN'
               pattern='^(?=(?:\D*\d){10}(?:(?:\D*\d){3})?$)[\d-]+$'
               onChange={({ target }) => setBooksData((prevIsbn) => ({ ...prevIsbn, Isbn: target.value }))}
+              disabled={!toggleAuthorForm && true }
```
Comment (SaleBojke) [inline src/components/modal/ModalCreateBooks.tsx:155]: ?

File: src/components/modal/ModalCreateBooks.tsx [hunk 7]
```diff
@@ -156,17 +161,24 @@ const ModalCreateBooks = ({ onClose, bookId } : Modal) => {
               className='input_for_quantity'
               placeholder='Quantity'
               onChange={({ target }) => setBooksData((prevQuantity) => ({ ...prevQuantity, Quantity: +target.value }))}
+              disabled={!toggleAuthorForm && true }
```
Comment (SaleBojke) [inline src/components/modal/ModalCreateBooks.tsx:164]: search this on whole project and replace

**Review by SaleBojke — APPROVED** [review]

File: src/components/bookDetails/BookDetails.tsx (hunk 3 above)
Comment (DragisaDraskovic) [reply src/components/bookDetails/BookDetails.tsx:98]: I fixed this.

File: src/components/bookDetails/BookDetails.tsx (hunk 2 above)
Comment (DragisaDraskovic) [reply src/components/bookDetails/BookDetails.tsx:44]: I fixed this.

File: src/components/bookDetails/BookDetails.tsx (hunk 4 above)
Comment (DragisaDraskovic) [reply src/components/bookDetails/BookDetails.tsx:148]: Ok i change logic

File: src/components/bookDetails/BookDetails.tsx (hunk 5 above)
Comment (DragisaDraskovic) [reply src/components/bookDetails/BookDetails.tsx:165]: Ok.

File: src/components/modal/ModalCreateBooks.tsx (hunk 6 above)
Comment (DragisaDraskovic) [reply src/components/modal/ModalCreateBooks.tsx:155]: I fixed this.

File: src/components/modal/ModalCreateBooks.tsx (hunk 7 above)
Comment (DragisaDraskovic) [reply src/components/modal/ModalCreateBooks.tsx:164]: I fixed this.

**Pull request merged**